src/models/index.js

```diff
@@ -0,0 +1,34 @@
+import { readdirSync } from 'fs';
+import { basename as _basename, join } from 'path';
+import Sequelize from 'sequelize';
+const basename = _basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(__dirname + '/../database/config.js')[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
```

Comment:
You won't need the last config parameter for this line since you're already passing a complete ConnectionURI:
https://sequelize.org/master/manual/getting-started.html (Check out the section: setting up a connection)

Reply:
The file was generated by `sequelize init` but let me go change that, thanks